packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -567,23 +575,18 @@ class ConfigScreenComponent extends React.Component<any, any> {
 					</div>
 				);
 			} else {
-				const onTextChange = (event: any) => {
-					updateSettingValue(key, event.target.value);
-				};
-
 				return (
 					<div key={key} style={rowStyle}>
 						<div style={labelStyle}>
 							<label>{md.label()}</label>
 						</div>
-						<input
-							type={inputType}
-							style={inputStyle}
-							value={this.state.settings[key]}
-							onChange={(event: any) => {
-								onTextChange(event);
-							}}
-							spellCheck={false}
+						<FontSearch
+							_key={key}
+							updateSettingValue={updateSettingValue}
+							inputType={inputType}
+							inputStyle={inputStyle}
+							settings={this.state.settings}
+							fonts={this.state.fonts}
```

Comment:
Doesn't it mean that all text input fields are replaced with a font search field?

Reply:
Oops, you are right. my bad
I have fixed it now.